docs/setup-cluster/k8s/custom-pod-specs.rst

```diff
@@ -7,7 +7,7 @@
 In a :ref:`Determined cluster running on Kubernetes <determined-on-kubernetes>`, tasks (e.g.,
 experiments, notebooks) are executed by launching one or more Kubernetes pods. You can customize
 these pods by providing custom `pod specs
-<https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#pod-v1-core>`__. Common use
+<https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#pod-v1-core>`__. Common use
```

Comment:
This page seems to be tied to specific Kubernetes versions. Gonna have to keep an eye on this one.